cli/cdsctl/project.go

```diff
@@ -63,3 +67,98 @@ func projectShowRun(v cli.Values) (interface{}, error) {
 	}
 	return *proj, nil
 }
+
+var projectCreateCmd = cli.Command{
+	Name:  "create",
+	Short: "Create a CDS project",
+	Args: []cli.Arg{
+		{Name: "project-key"},
+		{Name: "project-name"},
+	},
+	OptionalArgs: []cli.Arg{
+		{Name: "group-name"},
+	},
+	Aliases: []string{"add"},
+}
+
+func projectCreateRun(v cli.Values) error {
+	proj := &sdk.Project{Name: v["project-name"], Key: v["project-key"]}
+	return client.ProjectCreate(proj, v["group-name"])
+}
+
+var projectDeleteCmd = cli.Command{
+	Name:  "delete",
+	Short: "Delete a CDS project",
+	Args: []cli.Arg{
+		{Name: "project-key"},
+	},
+	Flags: []cli.Flag{
+		{
+			Name:  "force",
+			Usage: "Use force flag to delete all project dependencies with project",
+			IsValid: func(s string) bool {
+				if s != "true" && s != "false" {
+					return false
+				}
+				return true
+			},
+			Default: "false",
+			Kind:    reflect.Bool,
+		},
+	},
+	Aliases: []string{"rm", "remove", "del"},
+}
+
+func projectDeleteRun(v cli.Values) error {
+	projKey := v["project-key"]
+	if v.GetBool("force") {
```

Comment:
please ask for confirmation, we have a lib for this in the cli package

Reply:
done